spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -190,6 +190,8 @@
     subject { presenter.total_viewable_works }
     let(:ability) { double(::Ability, user_groups: ['public'], current_user: user) }
     let(:user) { create(:user) }
+    let(:collection) { FactoryBot.create(:collection_lw) }
```

Comment:
Same here RE: `collection_lw`.  Should it be...

```suggestion
    let(:collection) { FactoryBot.create(:hyrax_collection) }
```